src/flatpakManifest.ts

```diff
@@ -452,7 +452,7 @@ export class FlatpakManifest {
             '--allow=devel',
             `--bind-mount=/run/user/${uid}/doc=/run/user/${uid}/doc/by-app/${appId}`,
             ...this.finishArgs(),
-            "--talk-name='org.freedesktop.portal.*'",
```

Comment:
I'm not sure if this broke anything. Nothing so far when I tried it.

The quotations were removed because it errors out with `error: Invalid dbus name 'org.freedesktop.portal.*'`